data.js

```diff
@@ -208,6 +208,53 @@ exports.data = [
             link: "https://www.wri.org/",
             label: "world-resource-institution",
         },
+        established: {
+            content: "1982",
+            updatedBy: "Andre", updatedAt: "2020-01-04",
+        },
+        kpis: {
+            content: "- Organizations using their Guidelines \n- Contracts signed \n- literature wrote \n- Press Mentions",
```

Comment:
Removed caps in sentences: `"- Organizations using their guidelines \n- Contracts signed \n- literature wrote \n- Press mentions",`

Reply:
if you compare my suggestion and the original, you'll see less capitals.

Reply:
@yeah800

Reply:
check